.storybook/recipes/CalendarCard/CalendarCard.module.css

```diff
@@ -0,0 +1,59 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #CALENDAR CARD
+\*------------------------------------*/
+
+/**
+ * Card recipe for displaying calendar data
+ */
+.calendar-card {
+  width: 16rem;
```

Comment:
Will these cards always be 16rem or will these depend on the layout that they sit in. Just flagging this as something to think about moving forward so this can be reused in other contexts if needed.

Reply:
if size variants are added to the base Card component then this CSS size can be rm'ed.

Reply:
I don't think we'd want to add size variants to the base `Card` component. This card is used within like a calendar horizontal timeline thing and I'd imagine that each one of those columns could differ and may not be `16rem` all the time.